subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/locking/DependencyLockingIntegrationTest.groovy

```diff
@@ -192,7 +188,8 @@ dependencies {
         fails 'dependencies'
 
         then:
-        failure.assertThatCause(containsText("Lock file contained 'org:bar:1.0' but it is not part of the resolved modules"))
+        failure.assertHasCause("Dependency lock for configuration 'lockedConf' is out of date:\n" +
+            "\tLocking constraint was not resolved: 'org:bar:1.0'")
```

Comment:
:o: The message is IMO too focuses on an implementation detail ("locking constraint") and should rather explain from a user point of view what is happening:

> Dependency graph doesn't contain 'org:bar:1.0' which used to be found in the lock file

or something similar.

Reply:
Changing